test/WebSites/BasicWebSite/Views/Home/JsonHelperWithSettingsInView.cshtml

```diff
@@ -3,7 +3,10 @@
 @using Newtonsoft.Json.Serialization
 @{
     var settings = JsonSerializerSettingsProvider.CreateSerializerSettings();
-    settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+    settings.ContractResolver = new DefaultContractResolver
```

Comment:
We limit the number of `JsonSerializerSettings` instances and recommend users do the same. But this particular code is already unrealistic. I'm therefore not going to make it stranger by adding a `static` field.